source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -7683,16 +7506,10 @@ export class SerializationContext extends BaseSerializationContext {
         forbiddenActionTypes: string[],
         allowFallback: boolean): Action | undefined {
 
-        let forbiddenTypes: string[] = [];
-        if (parent.isInCarouselCard())
-        {
-            forbiddenTypes = Array.from(GlobalRegistry.forbiddenCarouselActions.values()).concat(forbiddenActionTypes);
-        }
```

Comment:
what is taking care of the forbidden items inside a carousel?

Reply:
as noted in the PR description, it's not implemented here yet...